javalib/src/main/scala/java/util/ArrayList.scala

```diff
@@ -66,7 +66,7 @@ class ArrayList[E] private (private[ArrayList] val inner: js.Array[E])
   override def addAll(index: Int, c: Collection[_ <: E]): Boolean = {
     c match {
       case other: ArrayList[_] =>
-        inner.splice(index, 0, other.inner: _*)
+        inner.splice(index, 0, other.inner.toSeq: _*)
```

Comment:
`other.inner` is only a `scala.collection.Seq`, not a `scala.collection.immutable.Seq`, hence the `toSeq` call.